docs-www/package.json

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "@brainhubeu/gatsby-docs-kit": "3.0.8",
     "@brainhubeu/react-carousel": "^1.13.44",
-    "gatsby": "2.20.10",
+    "gatsby": "2.20.24",
```

Comment:
Should we update `gatsby` in this MR? Is this necessary?

Reply:
I could not get `docs-www` to work with the Gatsby version we had, I bumped it to allow running it